storage/grpc_client.go

```diff
@@ -1511,7 +1511,10 @@ func (w *gRPCWriter) startResumableUpload() error {
 func (w *gRPCWriter) queryProgress() (int64, error) {
 	var persistedSize int64
 	err := run(w.ctx, func() error {
-		q, err := w.c.raw.QueryWriteStatus(w.ctx, &storagepb.QueryWriteStatusRequest{UploadId: w.upid})
+		q, err := w.c.raw.QueryWriteStatus(w.ctx, &storagepb.QueryWriteStatusRequest{
+			UploadId:                  w.upid,
+			CommonObjectRequestParams: toProtoCommonObjectRequestParams(w.encryptionKey),
```

Comment:
Why does this RPC need encryptionKey?

Reply:
TBH IDK but it has the parameters. I can exclude it, but I figured I'd err on the side of caution. I can remove it though, wdyt?

Reply:
I've removed it, its just silly atm.